proto/observer/query.proto

```diff
@@ -211,6 +216,15 @@ message QueryAllBlameRecordsResponse {
   repeated Blame blame_info = 1;
 }
 
+message QueryBlameByChainAndNonceRequest {
+  uint32 chain_id = 1;
```

Comment:
can we unify chain_id type as `int64` like other in other `query.proto` files? The `QueryProveRequest` one was was already updated in one of my pending PRs.